yt_dlp/extractor/prankcast.py

```diff
@@ -64,3 +64,71 @@ def _real_extract(self, url):
             'categories': [json_info.get('broadcast_category')],
             'tags': try_call(lambda: json_info['broadcast_tags'].split(','))
         }
+
+
+class PrankCastPostIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?prankcast\.com/[^/?#]+/posts/(?P<id>\d+)-(?P<display_id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://prankcast.com/devonanustart/posts/6214-happy-national-rachel-day-',
+        'info_dict': {
+            'id': '6214',
+            'ext': 'mp3',
+            'title': 'Happy National Rachel Day!',
+            'display_id': 'happy-national-rachel-day-',
+            'timestamp': 1704333938,
+            'uploader': 'Devonanustart',
+            'channel_id': 4,
+            'duration': 13175,
+            'cast': ['Devonanustart'],
+            'description': '',
+            'categories': ['prank call'],
+            'tags': None,
+            'upload_date': '20240104'
+        }
+    }, {
+        'url': 'https://prankcast.com/despicabledogs/posts/6217-jake-the-work-crow-',
+        'info_dict': {
+            'id': '6217',
+            'ext': 'mp3',
+            'title': 'Jake the Work Crow!',
+            'display_id': 'jake-the-work-crow-',
+            'timestamp': 1704346592,
+            'uploader': 'despicabledogs',
+            'channel_id': 957,
+            'duration': 263.287,
+            'cast': ['despicabledogs'],
+            'description': 'https://imgur.com/a/vtxLvKU',
+            'categories': [],
+            'tags': None,
+            'upload_date': '20240104'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id, display_id = self._match_valid_url(url).group('id', 'display_id')
+
+        webpage = self._download_webpage(url, video_id)
+        post = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['ssr_data_posts']
+        content = self._parse_json(post['post_contents_json'], video_id)[0]
+
+        uploader = post.get('user_name')
+        guests_json = self._parse_json(content.get('guests_json') or '{}', video_id)
```

Comment:
we should make this completely non-fatal
```suggestion
        guests_json = traverse_obj(content, ('guests_json', {json.loads}, {dict})) or {}
```